README.md

```diff
@@ -6,10 +6,29 @@ This project is a set of training materials and labs for a "Getting Started" lev
 
 ## How to use
 
-There are two sets of materials available
+To begin, select a recent [release](https://github.com/openmainframeproject/cobol-programming-course/releases) of the course. Under a given release, you should see two assets.
 
-- [Chapters](https://github.com/openmainframeproject/cobol-programming-course/blob/master/COBOL%20Programming%20with%20VSCode.pdf) which contain the lessons, broken up into chapters.
-- [Labs](https://github.com/openmainframeproject/cobol-programming-course/tree/master/Labs) which contain the source code used in the chapters.
+- The COBOL Programming with VSCode PDF asset contains the course instructions.
+- The Source code asset contains everything that was in the project when the release was published. Once you unzip this folder, notice the Labs folder. This folder contains source code used in the course.
```

Comment:
I think some newcomers may be confused by the source code in the release.
They may be used to download source code and compile/interpret it on their machines. In this case, the code from this repository is no needed to be downloaded, since the environment provided by IBM already contain the necessary datasets.

Is it expected that this course is attached to the environment provided by IBM? If not, should instructions to upload the code to their available mainframes be included?
If they have access to a mainframe but, without ZOWE, should instructions to upload and use the code using an x320 emulator be included?

Reply:
I like this suggestion - to provide instructions on how to setup the course using your own environment if desired.

Reply:
I updated the README to include the option to use your own environment. If detailed setup instructions were to be added in a future PR, I would recommend linking to that document from the README.